Libraries/Text/Text.js

```diff
@@ -154,7 +158,7 @@ const Text = React.createClass({
                 }
               }
               this.touchableHandleActivePressIn = () => {
-                if (this.props.suppressHighlighting || !this.props.onPress) {
+                if (this.props.suppressHighlighting || !this.props.onPress && !this.props.onLongPress) {
```

Comment:
I find this condition a bit hard to read. Can you add brackets to show how this logically belongs together or maybe split it into two return statements?